ush/parsing_namelists_FV3.sh

```diff
@@ -54,7 +54,7 @@ if [[ -n "${AERO_FIELD_TABLE:-}" ]]; then
     tail -n $(( nrec - prec )) "${FIELD_TABLE}" ; \
   } > field_table
   # add non-prognostic tracers from additional table
-  dnats=$(( dnats + dnats_aero ))
+  dnats=$(( dnats + ${dnats_aero:0} ))
```

Comment:
Is this default required?  
This is in the block of the condition `if [[ -n ${AERO_FIELD_TABLE} ]]`.  Should the check on line 43 also have a `DO_AERO = YES`?  If `DO_AERO` is `NO`, this will never be executed.  If it is `YES`, `dnats_aero` is defined in `config.aero` and if it isn't, it should be an error here.

Reply:
Thought it was needed, but now I think I was just missing exports.